deploy/examples/create-test-cluster.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+#set -eEx
+KUBECTL="minikube kubectl --"
+ROOK_EXAMPLES_DIR="."
+
+wait_for_ceph_cluster() {
+    echo "Waiting for ceph cluster"
+    WAIT_CEPH_CLUSTER_RUNNING=20
+    until [[ $(kubectl -n rook-ceph exec -it deploy/rook-ceph-tools -- ceph status) =~ .*'health: HEALTH_OK'.* ]]; do
+	echo "Waiting for Ceph cluster installed"
+	sleep ${WAIT_CEPH_CLUSTER_RUNNING}
+    done
+    echo "Ceph cluster installed and running"
+}
+
+show_info() {
+    DASHBOARD_PASSWORD=$($KUBECTL -n rook-ceph get secret rook-ceph-dashboard-password -o jsonpath="{['data']['password']}" | base64 --decode && echo)
+    IP_ADDR=$($KUBECTL get po --selector="app=rook-ceph-mgr" -n rook-ceph --output jsonpath='{.items[*].status.hostIP}')
+    PORT="$($KUBECTL -n rook-ceph -o=jsonpath='{.spec.ports[?(@.name == "dashboard")].nodePort}' get services rook-ceph-mgr-dashboard-external-http)"
+    BASE_URL="http://$IP_ADDR:$PORT"
+    echo "==========================="
+    echo "IP_ADDRES: $BASE_URL"
+    echo "PASSWORD: $DASHBOARD_PASSWORD"
+    echo "==========================="
+}
+
+setup_minikube_env() {
+    echo "Setting up minikube env"
+    minikube delete
+    minikube start --disk-size=40g --extra-disks=1 --driver kvm2
```

Comment:
should specify the os, as this would only works with linux

Reply:
I think we can provide the installation for different env https://rook.io/docs/rook/v1.12/Contributing/development-environment/#minikube

Maybe be auto detect if possible